testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/InitialAccessTokenTest.java

```diff
@@ -70,28 +65,6 @@ public void create() throws ClientRegistrationException, InterruptedException {
         }
     }
 
-    @Test
```

Comment:
I am not sure why this test was removed in this PR?

Reply:
Initial Token became signed in HS256 (or HS384, HS512).
Therefore, I thought that this test became meaningless.

Reply:
AFAIR the point of the test is to doublecheck that even if you change the signature algorithms for tokens, the initialToken is not affected and still signed with the HS256. So I would prefer to keep this test.

Reply:
Yes, I see. I will restore this test. Should I add new same test for PS256 signing?

Reply:
@tnorimat IMO it is fine to keep just the ES256 test. There is always some value in introducing new test, however there are also some not-good side effects to it (EG. testsuite running for longer time and consume more resources etc). So in this case, my vote is to just keep the current test :)

Reply:
OK. I will just only restore this current test.

Reply:
I've restored the test, and squashed commits.